src/cmd/manage_nsfs.js

```diff
@@ -563,7 +574,7 @@ async function manage_account_operations(action, data, show_secrets, user_input)
     } else if (action === ACTIONS.STATUS) {
         await get_account_status(data, show_secrets);
     } else if (action === ACTIONS.UPDATE) {
-        await update_account(data);
+        await update_account(data, user_input);
```

Comment:
I can pass the parsed flag, but I think it would not look good to pass only this flag to this function... (the function signature will be `async function update_account(data, is_flag_iam_operate_on_root_account)`.

The unset is on the data object, in `fetch_account_data`.
Then the value of it will be false `get_boolean_or_string_value` last condition.
So I can do it if we want.

Reply:
As discussed I will remove the unset with `''` for this variable and pass this variable only in `update_account` (instead of the `user_input`).